packages/css/src/components/anchor/anchor.template.ts

```diff
@@ -0,0 +1,54 @@
+import { Anchor } from '@dso-toolkit/sources';
+import { html, nothing } from 'lit-html';
+import { ifDefined } from 'lit-html/directives/if-defined';
+
+import { iconTemplate } from '../icon/icon.template';
+
+export function anchorTemplate({ icon, iconAfter, iconOnly, label, modifier, url }: Anchor) {
+  return html`
+    <a href=${url}
+       title=${ifDefined(iconOnly ? label : undefined)}
+       class=${ifDefined(modifier)}
+       target=${ifDefined(modifier && modifier.indexOf('extern') > -1 ? '_blank' : undefined)}
+       rel=${ifDefined(modifier && modifier.indexOf('extern') > -1 ? 'noopener noreferrer' : undefined)}
+    >
+      ${icon && !iconOnly
+        ? html`
+          ${iconTemplate({ icon: icon })}
+          ${!modifier || modifier.indexOf('btn') === -1
+            ? html` `
+            : nothing
+          }
+        `
+        : nothing
+      }
+
+      ${modifier && modifier.indexOf('btn') > -1
+        ? html`
+          <span class=${ifDefined( iconOnly ? 'sr-only' : undefined )}>${label}</span>
+        `
+        : html `
+          ${label}
```

Comment:
Hier hoef je geen `html` tagged template voor te openen

Reply:
En volgens mij bij die spatie in
```
html` `
```

ook niet

Reply:
Wat vul je daar in dan? gewoon een spatie op de plek waar nu html` ` staat? Ik denk dat mijn formatter zoiets automatisch weer weg haalt.